### DISCUSSION TITLE
Option to disable MustRevalidate=true

### DISCUSSION THREAD

**Discussion by yringler** [SixLabors/ImageSharp.Web#157] [Ideas]
Long story short - why is cache header revalidate always set to true? https://github.com/SixLabors/ImageSharp.Web/blob/d0559520245dffc484cd7fc77ae35fd20f0ff3b9/src/ImageSharp.Web/Middleware/ImageContext.cs#L203

Should there be a config for that?

**Comment by JimBobSquarePants** [comment]
You can already update the response headers before a response is sent via `ImageSharpMiddlewareOptions.OnPrepareResponseAsync`.

https://github.com/SixLabors/ImageSharp.Web/blob/d0559520245dffc484cd7fc77ae35fd20f0ff3b9/src/ImageSharp.Web/Middleware/ImageSharpMiddlewareOptions.cs#L149-L166

P.S. ImageSharp Discussions is the wrong discussion channel. I've migrated this for you to the correct one.
